strassengezwitscher/events/tests/test_api_views.py

```diff
@@ -132,3 +135,31 @@ def test_json_detail_events_incorrect(self):
         url = '/api/facebook1.json'
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+    """
+    Test backend filters
+    """
```

Comment:
An der Stelle sind Docstrings nicht passend. Wenn du die folgenden Method inhaltlich abtrennen willst, dann mach das am besten mittels eines normalen Kommentars, vielleicht auch mit vielen `#` hintereinander, wenn angebracht. Ansonsten könntest du das natürlich auch in eine separate Klasse verschieben, wobei ich das innerhalb von `EventAPIViewTests` schon passend finde.